src/Sylius/Component/Review/Factory/ReviewFactoryInterface.php

```diff
@@ -21,17 +22,17 @@
 interface ReviewFactoryInterface extends FactoryInterface
 {
     /**
-     * @param mixed $subjectId
+     * @param ReviewableInterface $subject
      *
      * @return ReviewInterface
      */
-    public function createForSubject($subjectId);
+    public function createForSubject(ReviewableInterface $subject);
```

Comment:
One question. We have two methods here `createForSubject` and `createForSubjectWithReviewer`. But the second parameter in a second method is optional. Are both of these method needed? Can we remove one of them? Or remove optional requirement in a second one?